serviceerror/multiOpAborted.go

```diff
@@ -0,0 +1,26 @@
+package serviceerror
+
+import (
+	failurepb "go.temporal.io/api/failure/v1"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var errMultiOperationAborted = MultiOperationAborted{}
+
+type MultiOperationAborted struct{}
```

Comment:
```suggestion
type MultiOperationAborted struct {
	Message string
	st      *status.Status
}
```

And allow message to be passed in constructor and make the pointer to this implement error instead of this whole. Benefits:

* This is used by SDKs too and should not swallow the
* Pointer receiver can carry more future information without copying regularly
* Status as a field means every invocation of `Status()` doesn't have to make a new one (it's usually meant to be a cheap getter IME)

If you want this to be a fixed package-level var on server side with a fixed string message and a fixed status, no prob. But don't bake it into the type used in SDK and server.

All of this applies to the other error too. Design for SDK and server use instead of just server use.

Reply:
I made all these choices intentionally because this particular error is different from the others in that it's meant to be a singleton/sentinel error, essentially. I don't see how any of these benefits will materialize for this particular error type in practice, but fair enough, I can align it with all the other types for consistency.

Reply:
I think when you align this object for client-side use, it'll make more sense to have these fields

Reply:
Okay, done!